code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -186,13 +186,15 @@
 	..()
 
 /mob/living/carbon/human/proc/Stasis(amount)
-	if((species.flags & NO_SCAN) || isSynthetic())
+	//if((species.flags & NO_SCAN) || isSynthetic()) vorestation removal
+	if(((species.flags & NO_SCAN) && !istype(species, /datum/species/xenochimera)) || isSynthetic()) //vorestation addition

```

Comment:
Too snowflake for me to include in this generic proc. If you want to have your species treated differently, do it on your species, not on this proc that affects everyone. You should just set in_stasis yourself if you want/need special treatment.